composer.json

```diff
@@ -76,14 +77,67 @@
     },
     "prefer-stable": true,
     "scripts": {
-        "analyse": "vendor/bin/phpstan analyse -c phpstan.neon -l max src",
-        "check-style": "vendor/bin/ecs check --ansi spec/ src/",
-        "fix-style": "vendor/bin/ecs check --ansi spec/ src/ --fix",
-        "phpspec": "vendor/bin/phpspec run --ansi",
-        "phpunit": "vendor/bin/phpunit",
-        "test": [
+        "all": [
+            "@checks",
+            "@tests"
+        ],
+        "analyse": [
+            "@ensure-test-container-exists",
+            "./vendor/bin/phpstan analyse -c phpstan.neon -l max src tests/Application/src"
+        ],
+        "assets": [
+            "@ensure-assets-installed",
+            "@ensure-assets-compiled"
+        ],
+        "behat": [
+            "SYMFONY_ENV=test composer ensure-database-ready",
+            "SYMFONY_ENV=test composer ensure-env-copied",
+            "./vendor/bin/behat --tags=\"~@javascript\" --no-interaction --format=progress"
+        ],
+        "check-style": "./vendor/bin/ecs check src tests/Application/src",
+        "checks": [
+            "composer validate --strict",
+            "composer normalize --dry-run",
+            "@check-style",
+            "@analyse"
+        ],
+        "ensure-assets-compiled": "[[ -d tests/Application/public/assets ]] || (cd tests/Application && yarn build && composer ensure-env-copied && bin/console assets:install public -e ${SYMFONY_ENV:-'dev'})",
+        "ensure-assets-installed": "[[ -d tests/Application/node_modules ]] || (cd tests/Application && yarn install)",
+        "ensure-database-created": "composer ensure-env-copied && (cd tests/Application && bin/console doctrine:database:create --if-not-exists -e ${SYMFONY_ENV:-'dev'})",
+        "ensure-database-ready": [
+            "@ensure-database-created",
+            "@ensure-schema-updated"
+        ],
+        "ensure-env-copied": "([[ ${SYMFONY_ENV:-'dev'} == 'dev' ]] && composer ensure-env-dev-copied) || ([[ ${SYMFONY_ENV:-'dev'} == 'test' ]] && composer ensure-env-test-copied) || echo 'Unknown environment ${SYMFONY_ENV}'",
+        "ensure-env-dev-copied": "(cd tests/Application && ([[ -f .env.dev.local ]] || cp .env  .env.dev.local))",
+        "ensure-env-test-copied": "(cd tests/Application && ([[ -f .env.test.local ]] || cp .env.test .env.test.local))",
+        "ensure-schema-updated": "composer ensure-env-copied && (cd tests/Application && bin/console doctrine:schema:update --force -e ${SYMFONY_ENV:-'dev'})",
+        "ensure-test-container-exists": "[[ -f tests/Application/var/cache/test/ApplicationTests_Setono_SyliusCalloutPlugin_Application_KernelTestDebugContainer.xml ]] || (cd tests/Application && php -d memory_limit=-1 bin/console cache:clear --env=test)",
```

Comment:
Test container isn't used by phpstan anymore. So the PHPStan Symfony extension can be removed and this line can be removed